examples/complete/fixtures.us-west-1.tfvars

```diff
@@ -0,0 +1,9 @@
+namespace = "eg"
```

Comment:
use `us-east-2` for all tests
https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment/tree/master/examples/complete

`us-west-1` is very limited and has different AZs names for diff AWS accounts

Also update the region in Go test